openapi/api.yml

```diff
@@ -302,20 +302,29 @@ paths:
 
   /regions:
     get:
-      summary: Returns a list of all distinct ITL regions as codes.
-      operationId: core.controllers.get_filters.get_regions
+      summary: Returns a list of all distinct ITL1 regions.
+      operationId: core.controllers.get_filters.get_geospatial_regions
       tags:
         - filters
       responses:
         '200':
-          description: The codes for all distinct ITL regions.
+          description: The codes and names for all distinct ITL1 regions.
           content:
             application/json:
               schema:
                 type: array
                 items:
-                  type: string
-                example: [ "TLC", "TLD", "TLK" ]
+                  type: object
+                  properties:
+                    name:
+                      type: string
+                    id:
+                      type: string
+                example:
+                  - name: "North West"
+                    id: "TLD"
+                  - name: "Scotland"
+                    id: "TLM"
```

Comment:
I can't see anywhere yet where we call this API endpoint externally - so we don't need to worry about backwards compatibility here, right?

Reply:
No exactly - it existed before but wasn't being called by the Find front end, which just populated the Region checkboxes from a hard-coded list. This API update is to make that checkbox list dynamic based on what's in the database. There was a unit test for this endpoint as well which I've updated as part of this PR to match how this now returns the data.